sw/device/tests/alert_handler_lpg_clkoff_test.c

```diff
@@ -612,5 +569,91 @@ bool test_main(void) {
     test_step_cnt++;
   }
 
+  /* TEST PHASE #2: TEST THE FATAL ALERTS (DV-only)
+
+    Use the same alert handler config from the previous phase:
+      - The timeout value is set to 256.
+      - All peripherals' alerts are enabled and locked.
+      - None of the alerts should trigger ping_timeout_alert
+
+    The SystemVerilog test sequence associated with this test is located here:
+      - hw/top_earlgrey/dv/env/seq_lib/chip_sw_alert_handler_lpg_clkoff_vseq.sv
+
+    TEST:
+    for_each_peripheral
+      1- Wait for random time
+      2- Enable the peripheral's clock
+      3- Trigger the peripheral's fatal alert
+      4- Disable the peripheral's clock
+      5- Wait for 1ms
```

Comment:
This is my mistake. Could you please change 1ms to 500us (based on the line 634).

Reply:
Done.